README.md

```diff
@@ -102,12 +102,18 @@ You can easily launch the whole transition system using Docker and thus not havi
 ### Building the image
 `docker build -t testtransition .`
 (You can replace testtransition with your prefered image name. Don't forget to update any other command and compose file if you do so)
-To run the application directly, you'll need to add an .env as previously described before building. 
+To run the application directly, you'll need to add an .env as previously described before building.
```

Comment:
Maybe we should say that these are some default, but we need to have a few mandatory one. 

We could also have a docker specific .env file.